src/clusterfuzz/_internal/issue_management/google_issue_tracker/client.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Gets a Google Issue Tracker HTTP client."""
+
+import google.oauth2.credentials
+import google_auth_httplib2
+import googleapiclient
+import httplib2
+
+_DISCOVERY_URL = "https://issuetracker.googleapis.com/$discovery/rest?version=v1"
+_O_AUTH_SCOPE = "https://www.googleapis.com/auth/buganizer"
+_REQUEST_TIMEOUT = 60
+HttpError = googleapiclient.errors.HttpError
```

Comment:
I don't think this is used anywhere. It will be when we make the real calls to public API but not yet.

Reply:
Looks like this is used in issue_tracker, in the method `_execute`, line 576

Reply:
Ah sorry, it did not show up in my search because the issue_tracker file had a "Large diffs are not rendered by default"